cmd/create/externalauthprovider/externalauthprovider_suite_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func TestIdp(t *testing.T) {
+func TestExternalAuthProvider(t *testing.T) {
```

Comment:
Not related, but I found that the test suite name is incorrect for external auth